project/plugins.sbt

```diff
@@ -2,6 +2,7 @@ addSbtPlugin("com.lightbend.paradox"     % "sbt-paradox"             % "0.9.2")
 addSbtPlugin("com.typesafe.sbt"          % "sbt-site"                % "1.4.1")
 addSbtPlugin("com.typesafe.sbt"          % "sbt-ghpages"             % "0.6.3")
 addSbtPlugin("org.typelevel"             % "sbt-typelevel-ci-release" % "0.4.17")
+addSbtPlugin("org.typelevel"             % "sbt-typelevel-mergify"   % "0.4.17")
```

Comment:
It's really this easy 😜 all the other changes in this PR were generated by the plugin.